com.unity.sg2/Editor/GraphDeltaRegistry/GraphDelta/GraphHandler.cs

```diff
@@ -159,12 +204,12 @@ public void ReconcretizeAll()
                         {
                             if (builder.GetRegistryFlags() == RegistryFlags.Func)
                             {
-                                ReconcretizeNode(node.ID.FullPath);
+                                ReconcretizeNodeNoPropagation(node.ID.FullPath);
                             }
 
                             if (builder.GetRegistryKey().Name == ContextBuilder.kRegistryKey.Name)
                             {
-                                ReconcretizeNode(node.ID.FullPath);
+                                ReconcretizeNodeNoPropagation(node.ID.FullPath);
                             }
                         }
                     }
```

Comment:
Combined changes for ReconcretizeAll() should be a bit of a perf bump for large graphs.